helm/release-operator-chart/values.yaml

```diff
@@ -1,3 +1,8 @@
+name: release-operator
```

Comment:
Why we don't do similarly to https://github.com/giantswarm/aws-operator/blob/dbb7c297657d782561f3dd83e567303aace22278/helm/aws-operator/values.yaml#L31-L39 ?

I.e. 

```
resource:
  default:
    name: "release-operator"
    namespace: "giantswarm"
  networkPolicy:
    name: "release-operator-network-policy"
    namespace: "giantswarm"
```

Reply:
I can do, i'm just trying not to scope my PRs too wide. will all operators eventually undergo VOO? Or do you mean just to keep things more consistent?

Reply:
This is the only one that won't so in this case it's consistency only. I'm not to fussed about other project because we will change them anyway during VOO flattening.